### ISSUE TITLE
crash a few seconds after startup

### ISSUE THREAD

**Issue by shocktail39** [vircadia/vircadia#1172]
version: 2021.1.1 appimage
os: arch linux
cpu: amd ryzen 7 3700x
gpu: asrock rx 580 (polaris 10)
hmd: valve index, steamvr beta 1.17.5
packages that might be relevant: vulkan-radeon 20.3.4-3, amdvlk 2021.Q1.6-1, vulkan-icd-loader 1.2.172-1, mesa 21.0.1-1, mesa-vdpau 21.0.1-1, libva-mesa-driver 21.0.1-1, xf86-video-amdgpu 19.1.0-2

crash message: Vircadia-2021.1.1-Eos.AppImage: /home/motofckr9k/Vircadia/source/libraries/gpu/src/gpu/Buffer.cpp:102: void gpu::Buffer::Update::apply() const: Assertion `buffer._applyUpdateCount == updateNumber' failed.

**Comment by JulianGro** [comment]
user haagch also had/has this issue and says "pretty sure it only happened when put off the headset"

**Comment by shocktail39** [comment]
does that mean it happens when they take it off of their head?  is it able to detect that?

**Comment by JulianGro** [comment]
Yes. When you take off your HMD Vircadia should automatically switch to Desktop mode and vice versa.

**Comment by shocktail39** [comment]
oh that's kind of cool.  as a temp workaround, do you think you can point me to where i can patch the code to disable this functionality?

**Comment by JulianGro** [comment]
No but you can put tape over the sensor on your Valve Index ;P

On another note: The line in that crash message hasn't been changed for three years. Last commit: https://github.com/vircadia/vircadia/commit/37fdf81f1bfd932ef471e7c5769e7f53376ed107

**Comment by shocktail39** [comment]
not too familiar with the hardware.  where are the relevant sensors located?

**Comment by JulianGro** [comment]
At the top between the lenses. It just looks like a black shiny plastic tab sticking out.

**Comment by shocktail39** [comment]
thanks found it.  will follow-up with if taping it down fixes the problem.

**Comment by shocktail39** [comment]
couldn't find tape, but turning up the time before the screen turns off from inactivity also seems to have mitigated it.  thank you for your help.

**Comment by daleglass** [comment]
Hello, could you get a backtrace of this one? Start the interface under gdb, then when it crashes, use the `bt` command.

**Comment by JulianGro** [comment]
> Hello, could you get a backtrace of this one? Start the interface under gdb, then when it crashes, use the `bt` command.

I can probably get it to crash on demand. So if someone specifically want's to work on this just tell me here or on Discord.
Just seems like too much of a hassle to do one right now, since I still have four other crash issues with backtraces open and I assume backtraces also get old.
